server.go

```diff
@@ -133,7 +132,7 @@ func (s *Server) initialize(interceptorChain *rpcperms.InterceptorChain) error {
 				Checkers: []macaroons.Checker{
 					macaroons.IPLockChecker,
 				},
-				RequiredPerms: perms.RequiredPermissions,
+				RequiredPerms: taprpc.RequiredPermissions,
```

Comment:
Have tried to get things building with _just_ the move to have `perms` be a sub-module? 

If for whatever reason that isn't sufficient, then I think in order to adopt this RPC sub-module, we'd want to relax some rules we have re replace directives during development. Otherwise every RPC related change will need a PR for updating the protos, then another one for using the new set of protos.